server/game/cards/13.3-PoS/JonConnington.js

```diff
@@ -13,7 +13,7 @@ class JonConnington extends DrawCard {
             message: '{player} uses {source} to reveal {target}',
             handler: context => {
                 this.game.resolveGameAction(
-                    GameActions.revealCard(context => ({ card: context.target })),
+                    GameActions.revealCards(context => ({ card: context.target })),
```

Comment:
Don't modify any of the existing `revealCard` cards, I'm updating them on my own branch.

Reply:
Reverted `Jon Connington` and `Highgarden` adjustments in 67ba4a5b9b3ac39ba257adcbc3de7d867395be2f.

I do recommend we keep it as the updated `GameActions.revealCards`, to keep things consistent; might be best to do that in your branch or whichever branch is being pushed last. 😄